src/entry.js

```diff
@@ -2,6 +2,7 @@ import Duration from 'duration'
 import shortid from 'shortid'
 import moment from 'moment'
 import parser from './parser'
+import conf from './config'
 
 export const hashPattern = /(#\w+[\w-]*)/g
 export const version = 4
```

Comment:
I thought about adding an upgrader to do that, but the problem is that if we add an upgrader the `.tickbinrc` file still doesn't contain the user ID, so we don't know what to add to the documents. They need to actually login either way unfortunately.